deploy.sh

```diff
@@ -16,7 +16,7 @@ Available environments:
 `( IFS=$'\n'; echo "${ENVIRONMENTS[*]}" )`
 
 Examples:
-  deploy staging example.dev
+  deploy staging staging.example.com
```

Comment:
So, this now will drop the `staging` subdomain and be...
`deploy staging example.com`